.travis.yml

```diff
@@ -9,10 +9,19 @@ env:
 install:
   - make image
   - docker images
+  - curl -L https://github.com/arminc/clair-scanner/releases/download/v8/clair-scanner_linux_amd64 > clair-scanner
+  - chmod +x clair-scanner
+  - sudo mv clair-scanner /usr/local/bin
   
 script:
-  - docker run -d -p 8080:8080 --name pega-ready pegasystems/pega-ready
+  - docker images
   - docker ps
+  - curl -LO https://storage.googleapis.com/container-structure-test/latest/container-structure-test-linux-amd64 && chmod +x container-structure-test-linux-amd64 && sudo mv container-structure-test-linux-amd64 /usr/local/bin/container-structure-test
+  - container-structure-test test --image pegasystems/pega-ready:qualitytest --config tests/pega-web-ready-testcases.yaml
+  - container-structure-test test --image pegasystems/pega-ready --config tests/pega-web-ready-release-testcases.yaml
```

Comment:
@dcasavant Hardcoded the repository name here as pegasystems/pega-ready as the travis repository specific variables were not configured. 
Can we configure REPO_NAME as well in the repository settings? so that we can use the below statements instead of direct values here.
container-structure-test test --image $(DOCKER_USER)/$(REPO_NAME) --config tests/pega-web-ready-release-testcases.yaml

Reply:
Yes, they should be configured.  If it is not working it may be a referencing format problem.